docs/VnVPlan/VnVPlan.tex

```diff
@@ -316,6 +316,1047 @@ \subsection{Tests for Nonfunctional Requirements}
+        \item Document any issues or inconsistencies found during testing.
+        \item Provide recommendations for improving accessibility features based on user feedback and observed performance.
+        \item Ensure that all accessibility options meet the fit criteria of having at least three color themes and supporting text size adjustments up to 200\% without loss of content or functionality.
+    \end{enumerate}
+\end{enumerate}
+
+\item{Keyboard Shortcuts Accessibility Test: T-UH9\\}
+
+Related Reqs: NFR-UH9
+
+Type: Manual, Usability
+
+Initial State: The platform includes keyboard shortcuts for all core actions such as navigating between tasks and submitting labels.
+
+Input/Condition: Users interact with the platform exclusively using keyboard navigation without the use of a mouse.
+
+How test will be performed:
+\begin{enumerate}
+    \item \textbf{Identify Core Actions:}
+    \begin{enumerate}
+        \item Compile a list of all core actions within the platform (e.g., navigating between tasks, submitting labels, accessing settings).
+        \item Document the corresponding keyboard shortcuts for each core action.
+    \end{enumerate}
+    \item \textbf{User Recruitment:}
+    \begin{enumerate}
+        \item Recruit a diverse group of participants, including power users and users with limited mobility who rely on keyboard navigation.
+    \end{enumerate}
+    \item \textbf{Task Execution:}
+    \begin{enumerate}
+        \item Instruct participants to perform a series of predefined tasks using only keyboard shortcuts.
+        \item Ensure that participants refrain from using a mouse or other pointing devices during the test.
+    \end{enumerate}
+    \item \textbf{Observation and Recording:}
+    \begin{enumerate}
+        \item Observe participants as they use keyboard shortcuts to perform each core action.
+        \item Record the success rate of accessing and completing each action via keyboard.
+    \end{enumerate}
+    \item \textbf{Data Analysis:}
+    \begin{enumerate}
+        \item Calculate the percentage of core actions that were successfully accessed and performed using keyboard shortcuts.
+        \item Verify that at least 95\% of core actions are accessible through keyboard navigation.
+    \end{enumerate}
+    \item \textbf{Reporting and Recommendations:}
+    \begin{enumerate}
+        \item Document any issues or shortcomings discovered during testing.
+        \item Provide recommendations for adding missing shortcuts or improving existing ones to enhance keyboard accessibility.
+        \item Ensure that all core actions meet the fit criterion of being accessible via keyboard shortcuts.
+    \end{enumerate}
+\end{enumerate}
+
+\end{enumerate}
+
+% NFRs 12---------------------------------------------------------
+\subsubsection{Performance Requirements}
+% 12.1
+\paragraph{Speed and Latency Tests}
+\begin{enumerate}
+\item{New User Account Processing Time Validation: T-PR0\\}
+
+Related Reqs: NFR-PR0
```

Comment:
I think this relates to both PR0 and PR1 correct?

Reply:
Yup